README.md

```diff
@@ -57,6 +57,8 @@ pip install 'crewai[tools]'
 
 ### 2. Setting Up Your Crew
 
+In order to run the following example you need to have installed the `crewai-tools` package.
```

Comment:
Isn't this covered here?  https://github.com/joaomdmoura/crewAI/pull/350/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R55

Reply:
indeed